### PR TITLE
[NO-JIRA] Add option to customize bottom sheet insets

### PR DESCRIPTION
**Pull request by jookes**
This PR introduces an extra `insets` parameter to the BottomSheet initialization call. This allows for greater customization. 

The defaults that were used in ModalBottomSheetLayout and PersistentBottomSheetLayout have been copied over to a new BottomSheetInset struct. 

+ [ ] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-ios/blob/master/CONTRIBUTING.md)

Remember to include the following changes:
+ [ ] `UNRELEASED.md`
+ [ ] `README.md`
+ [ ] Tests
+ [ ] [Screenshotting code](https://github.com/Skyscanner/backpack-ios/blob/master/Example/Backpack%20Screenshot/Screenshots.swift)
+ [ ] Adding a component? Remember to expose it in the [main `Backpack.h` header file](https://github.com/Skyscanner/backpack-ios/tree/master/Backpack/Backpack.h)
+ [ ] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

_If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/master/CODE_REVIEW_GUIDELINES.md)_

### CODE REVIEW

File: Example/Backpack.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,6 +7,14 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+D28F6B8923A1091D001A188F /* SkyscannerRelativeiOS-Black.ttf in Resources */ = {isa = PBXBuildFile; fileRef = D28F6B8123A1091D001A188F /* SkyscannerRelativeiOS-Black.ttf */; };
```
Comment (georgegillams) [inline Example/Backpack.xcodeproj/project.pbxproj:10]: Could this be reverted plz?

File: Example/Backpack.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -234,6 +242,14 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+D28F6B8123A1091D001A188F /* SkyscannerRelativeiOS-Black.ttf */ = {isa = PBXFileReference; lastKnownFileType = file; path = "SkyscannerRelativeiOS-Black.ttf"; sourceTree = "<group>"; };
```
Comment (georgegillams) [inline Example/Backpack.xcodeproj/project.pbxproj:245]: Same as above

**Review by georgegillams — CHANGES_REQUESTED** [review]
Sorry for the delay. Looks good to me except a couple of files that need reverting 👍

File: Example/Backpack.xcodeproj/project.pbxproj (hunk 1 above)
Comment (jookes) [reply Example/Backpack.xcodeproj/project.pbxproj]: Reverted!

File: Example/Backpack.xcodeproj/project.pbxproj (hunk 2 above)
Comment (jookes) [reply Example/Backpack.xcodeproj/project.pbxproj]: Reverted!

**Review by georgegillams — APPROVED** [review]

**Pull request merged**